docs/docs/architecture/notes.md

```diff
@@ -18,47 +17,47 @@ As shown in the above picture:
 
 Note scripts are created together with their inputs, i.e., the creator of the note defines which inputs are used at note execution by the executor. However, the executor or prover can pass optional note args. Note args are data put onto the the stack right before a note script is executed. These are different from note inputs, as the executing account can specify arbitrary note args.
 
-There exist [standard note scripts](https://github.com/0xPolygonMiden/miden-base/tree/main/miden-lib/asm/note_scripts) (P2ID, P2IDR, SWAP) that users can create and add to their notes using the [Miden client](../network/miden-clients.md) or by calling internal [Rust code](https://github.com/0xPolygonMiden/miden-base/blob/fa63b26d845f910d12bd5744f34a6e55c08d5cde/miden-lib/src/notes/mod.rs#L15-L66).
+There exist [standard note scripts](https://github.com/0xPolygonMiden/miden-base/tree/main/miden-lib/asm/note_scripts) (P2ID, P2IDR, SWAP) that users can create and add to their notes using the [Miden client]() or by calling internal [Rust code](https://github.com/0xPolygonMiden/miden-base/blob/fa63b26d845f910d12bd5744f34a6e55c08d5cde/miden-lib/src/notes/mod.rs#L15-L66).
 
 * P2ID and P2IDR scripts are used to send assets to a specific account ID. The scripts check at note consumption if the executing account ID equals the account ID that was set by the note creator as note inputs. The P2IDR script is reclaimable and thus after a certain block height can also be consumed by the sender itself. 
 * SWAP script is a simple way to swap assets. It adds an asset from the note into the consumer's vault and creates a new note consumable by the first note's issuer containing the requested asset. 
 
-### Example note script Pay to ID (P2ID)
+#### Example note script Pay to ID (P2ID)
 <details>
   <summary>Want to know more how to ensure a note can only be consumed by a specified account?</summary>
 
-  #### Goal of the P2ID script
+  ##### Goal of the P2ID script
   The P2ID script defines a specific target account ID as the only account that can consume the note. Such notes ensure a targeted asset transfer. 
 
-  #### Imports and context
+  ##### Imports and context
   The P2ID script uses procedures from the account, note and wallet API.
   ```
   use.miden::account
   use.miden::note
   use.miden::contracts::wallets::basic->wallet
   ```
-  As discussed in detail in [transaction kernel procedures](../transactions/transaction-procedures.md) certain procedures can only be invoked in certain contexts. The note script is being executed in the note context of the [transaction kernel](../transactions/transaction-kernel.md).
+  As discussed in detail in [transaction kernel procedures](../transactions/transaction-procedures.md) certain procedures can only be invoked in certain contexts. The note script is being executed in the note context of the [transaction kernel](transactions/transaction-kernel.md).
```

Comment:
Does this link work? `[transaction kernel](transactions/transaction-kernel.md)`

Isn't the transaction kernel a subchapter of transaction so one layer down? Or is this different in mkdocs?

Reply:
all the transaction docs, including the introduction, are in the same dir